gnovm/pkg/gnolang/store.go

```diff
@@ -339,6 +327,13 @@ func (ds *defaultStore) GetPackage(pkgPath string, isImport bool) *PackageValue
 	return nil
 }
 
+// NOTE: Does not consult pkgGetter, used only for
+// getting *PackageValue from store.
+func (ds *defaultStore) GetPackageFromStore(oid ObjectID) *PackageValue {
+	obj := ds.GetObject(oid)
+	return obj.(*PackageValue)
+}
```

Comment:
what about using pkgID instead and create the oid from it. (new_time = 1)
this way, we can easily use this function to retrieve package with any object containing the pkgID ?